<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-SGV8PKT" failures="1" tests="11" name="homePage.myTestcases" time="21.654" errors="0" timestamp="2024-08-23T19:06:39 EET" skipped="0">
  <testcase classname="homePage.myTestcases" name="CheckTheDefaultLangugeIsEnglish" time="0.085"/>
  <system-out/>
  <testcase classname="homePage.myTestcases" name="CheckdefaultCurrency" time="0.078"/>
  <system-out/>
  <testcase classname="homePage.myTestcases" name="CheckContactNumber" time="0.053"/>
  <system-out/>
  <testcase classname="homePage.myTestcases" name="CheckQitagLogo" time="0.209"/>
  <system-out/>
  <testcase classname="homePage.myTestcases" name="TestHotelTabIsNotSelected" time="0.050"/>
  <system-out/>
  <testcase classname="homePage.myTestcases" name="CheckDepatureDate" time="0.565"/>
  <system-out/>
  <testcase classname="homePage.myTestcases" name="RandomlyChangeTheLanguage" time="0.767"/>
  <system-out/>
  <testcase classname="homePage.myTestcases" name="FillHotelTab" time="1.204"/>
  <system-out/>
  <testcase classname="homePage.myTestcases" name="RandomlySelectTheNumberOfVistor" time="4.784"/>
  <system-out/>
  <testcase classname="homePage.myTestcases" name="CheckThePageFullyLoaded" time="12.997"/>
  <system-out/>
  <testcase classname="homePage.myTestcases" name="SortItemsLowestToHighestPrice" time="0.862">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:819)
at org.testng.Assert.assertEquals(Assert.java:829)
at homePage.myTestcases.SortItemsLowestToHighestPrice(myTestcases.java:211)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- SortItemsLowestToHighestPrice -->
  <system-out/>
</testsuite> <!-- homePage.myTestcases -->
